pandas/tests/arrays/sparse/test_array.py

```diff
@@ -1128,6 +1128,13 @@ def test_from_coo(self):
         expected = pd.Series([4, 9, 7, 5], index=index, dtype='Sparse[int]')
         tm.assert_series_equal(result, expected)
 
+    @td.skip_if_no_scipy
```

Comment:
Is this test different from the other one? I think we only need the one in test_accessor.py